File: php/Produk.php
<?php

class Produk
{
    private $nama;
    private $harga;
    private $merek;

    public function __construct($nama = "", $harga = "", $merek = "")
    {
        $this->nama = $nama;
        $this->harga = $harga;
        $this->merek = $merek;
    }

    public function getNama()
    {
        return $this->nama;
    }

    public function setNama($nama)
    {
        $this->nama = $nama;
    }

    public function getHarga()
    {
        return $this->harga;
    }

    public function setHarga($harga)
    {
        $this->harga = $harga;
    }

    public function getMerek()
    {
        return $this->merek;
    }

    public function setMerek($merek)
    {
        $this->merek = $merek;
    }
}

?>

File: python/Produk.py
class Produk:
    def __init__(self, nama = "", harga = "", merek = ""):
        self.__nama = nama
        self.__harga = harga
        self.__merek = merek

    def getNama(self):
        return self.__nama

    def setNama(self, nama):
        self.__nama = nama

    def getHarga(self):
        return self.__harga

    def setHarga(self, harga):
        self.__harga = harga

    def getMerek(self):
        return self.__merek

    def setMerek(self, merek):
        self.__merek = merek
        